tests/kernel/boot_page_table/src/boot_page_table.c

```diff
@@ -82,7 +82,22 @@ static void ending_start_addr_range(u32_t start_addr_range)
 			      addr_range, REGION_PERM);
 	}
 }
+/**
+ * @brief Test for boot page table
+ * @defgroup kernel_boot_page_table boot_page_table_tests
```

Comment:
`@defgroup kernel_boot_page_table_tests Boot Page Table`

We probably want to put this in another category with other tests.